Tools/Pipeline/Windows/MainView.cs

```diff
@@ -77,6 +78,9 @@ public void Attach(IController controller)
             _controller.OnProjectLoading += invokeUpdateMenus;
             _controller.OnProjectLoaded += invokeUpdateMenus;
 
+            _controller.OnProjectLoading += OnProjectLoading;
+            _controller.OnProjectLoaded += OnProjectLoaded;
```

Comment:
My own preference is to avoid using lambas in almost all situations. But I can change it if you want.